DriverLoader.cpp

```diff
@@ -37,6 +37,10 @@
 #define WIN32_LEAN_AND_MEAN
 #define NO_MINMAX
 #include <windows.h>
+#elif defined(OSVR_LINUX)
+#include <dlfcn.h>
+#elif defined(OSVR_MACOSX)
+#include <dlfcn.h>
```

Comment:
Is it worth having the same code in separate `OSVR_LINUX` and `OSVR_MACOSX` branches or should they be conflated?